docs/user/pings/events.md

```diff
@@ -4,6 +4,8 @@
 The events ping's purpose is to transport all of the event metric information.
 If the application crashes, an `events` ping is generated next time the application starts with events that were not sent before the crash.
 
+The `events` ping is not sent when using the Glean Python bindings.
```

Comment:
Mh, why is that? Is it because it's not implemented?

Reply:
Well, yes, right now we don't have events.  But even once that's done, it's sent when going to background (which doesn't make sense in non-mobile), and there's no way to send built-in pings.

Reply:
They're also sent at startup and whenever we reach the cap (500 events), but I agree the 'background' part doesn't make sense. Since this is more about defining a new behaviour for this ping, I'd recommend filing a bug and mentioning here that "send on background" is not supported when using Python bindings.

Reply:
That makes sense.